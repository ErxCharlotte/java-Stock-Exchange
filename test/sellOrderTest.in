ADD trader1 100.00
ADD trader2 100.00
IMPORT trader1 ABC 100
BUY trader2 ABC 200 20
SELL trader1 ABC 100 10
EXIT