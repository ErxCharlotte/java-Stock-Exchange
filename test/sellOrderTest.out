$ Success.
$ Success.
$ Trader now has 100.00 units of ABC.
$ No trades could be made, order added to buy book.
$ Product sold in entirety, trades as follows:
trader1->trader2: 100.00xABC for $20.00.
$ Have a nice day.